sycl/test-e2e/KernelFusion/queue-shortcut-functions.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %{build} %{embed-ir} -o %t.out
+// RUN: %{build} %{embed-ir} -fsycl-decompose-functor -o %t.out
```

Comment:
I'm assuming passing USM pointers won't work by default on SYCL fusion after this change on NVPTX targets, right?

Reply:
The patch prevents the break down of structs by default where it can, so fusion with USM would require passing this flag.